webapps/density-custom-chart/webapp.json

```diff
@@ -0,0 +1,39 @@
+{
+    "meta": {
+        "label": "Density map",
+        "description": "Generate a density map from geospatial data. Takes as input WKT points.",
+        "icon": "icon-globe"
+    },
+
+    "baseType": "STANDARD",
+    "hasBackend": "true",
+
+    "chart" : {
+        "datasetParamName": "dataset",
+        "leftBarLabel": "Chart parameters",
+        "leftBarParams": [],
+        "leftBarTemplate": "leftBarParamsTemplate.html",
+        "leftBarModule": "geospatial.density",
+        "topBar": "STD_FORM",
+        "topBarParams": [
```

Comment:
Indeed, this point was also noticed during product oriented sessions but we decided to leave it as is since we plan to make this chart part of the native ones